core/tests/puppeteer/lighthouse_setup.js

```diff
@@ -50,6 +50,8 @@ var createTopicButtonSelector = '.protractor-test-create-topic-button';
 var topicNameField = '.protractor-test-new-topic-name-field';
 var topicUrlFragmentField = '.protractor-test-new-topic-url-fragment-field';
 var topicDescriptionField = '.protractor-test-new-topic-description-field';
+// eslint-disable-next-line max-len
```

Comment:
Is there a way to fix this other than disabling the linter?

Reply:
I have shortened the variable name. PTAL.